action.yml

```diff
@@ -190,6 +190,12 @@ inputs:
       then one can specify 'gitHubEnterprise'.
     required: false
     default: 'gitHub'
+  pr-comment-github-token:
+    description: >
+      The Github Token for the repo that will be used to add a comment to the PR.
+      The comment will consist of the output of the notebook.
```

Comment:
To my earlier comment - what permissions does this token need / could we provide a docs link? Does using `GITHUB_TOKEN` work?

Reply:
using github token does work. I tested this with the provided by default env variable `${{gtihub.token}}` and the action was able to comment on a PR. Hence the doc of the param:
```
To use the default github token provided, just set the value of this input to the env variable 'github.token'.
```

Reply:
I will link this in the docs: https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token that the `${{ secrets.GITHUB_TOKEN }}` must have `Default access
(permissive)` permission for action to comment on PR